lib/core/src/Cardano/Wallet/Api/Server.hs

```diff
@@ -1997,8 +1997,10 @@ showT = T.pack . show
 instance LiftHandler ErrCurrentEpoch where
     handler = \case
         ErrUnableToDetermineCurrentEpoch ->
-            apiError err500 UnableToDetermineCurrentEpoch
-                "I'm unable to determine the current epoch."
+            apiError err500 UnableToDetermineCurrentEpoch $ mconcat
+                [ "I'm unable to determine the current epoch. "
+                , "Please wait a while for the node to sync and try again."
```

Comment:
Since it's unexpected for a node client to not know which epoch they are in at any time, it might be helpful to add a little detail as to why this cannot be known. If it's too complex to provide over an error message, maybe a link to some docs would be something to add later when available. I assume this is a common issue for chain-sync clients.

Reply:
> If it's too complex to provide over an error message, maybe a link to some docs would be something to add later when available.

This sounds like a sensible suggestion. :+1: 

From conversations with @Anviking, it seems that this situation will only occur very rarely, around the time of a hard fork, and when a node is not yet fully in sync. I think if we do start seeing reports of this error in the wild, we can look into adding more information to the error.